src/ir/comp.rs

```diff
@@ -434,6 +445,11 @@ impl CompInfo {
         &self.constructors
     }
 
+    /// Get this type's destructor.
+    pub fn destructor(&self) -> &Option<ItemId> {
+        &self.destructor
```

Comment:
I don't see any reason to avoid copies here; it'll just make using this slightly more annoying. Instead we should do:

```rust
pub fn destructor(&self) -> Option<ItemId> {
        self.destructor
}
```

since `Option<T>` is `Copy` if `T` is `Copy`.